.gitignore

```diff
@@ -3,3 +3,4 @@
 /.pids/
 
 __pycache__/
+/.mypy_cache/
```

Comment:
Was this necessary before as well?

Reply:
Sort of. The repo never runs MyPy, but I've been running it locally through system installed MyPy and kept accidentally adding the cache to Git.

Do you prefer I remove this for a cleaner diff?

Reply:
If it's not going to be generated for all users of the repo / we never run the mypy command in any scripts in this repo then I might suggest considering [`.git/info/exclude`](https://stackoverflow.com/questions/22906851/when-would-you-use-git-info-exclude-instead-of-gitignore-to-exclude-files) -- I use it for e.g. `.dir-locals.el` files which emacs uses.